chainer/iterators/sequential_iterator.py

```diff
@@ -0,0 +1,68 @@
+from __future__ import division
+
+from chainer.dataset import iterator
+
+
+class SequentialIterator(iterator.Iterator):
+
+    """Dataset iterator that sequentailly yields the examples.
+
+    This is a simple implementation of :class:`~chainer.dataset.Iterator`
+    that just visits each example in the order of indexes. In general, this
+    should not be used for training, but it is useful for validation. It is
+    also useful when the order of examples is important and should not be
+    broken.
+
+    Args:
```

Comment:
Thank you, I agree with you. I updated the code and merge these iterators into `SerialIterator`.